### DISCUSSION TITLE
Bluetooth Version Finder

### DISCUSSION THREAD

**Discussion by wincmd64** [turkaysoft/glow#14] [Ideas]
Pls add Bluetooth version display like in https://www.sordum.org/10772/bluetooth-version-finder-v1-4/

**Comment by turkaysoft** [comment]
> Pls add Bluetooth version display like in https://www.sordum.org/10772/bluetooth-version-finder-v1-4/

Thank you for your suggestion.
I will be sure to add it if I can figure out how to correctly pull the Bluetooth version from the system.

**Reply by wincmd64** [reply]
https://support.microsoft.com/en-us/windows/what-bluetooth-version-is-on-my-pc-f5d4cff7-c00d-337b-a642-d2d23b082793

**Reply by turkaysoft** [reply]
Unfortunately, it is not as simple as it looks there to pull with C#. It would have been much easier if I was using UWP instead of WinForm, but making a System Analysis Software with WinForm is much more difficult than making a normal software.

**Reply by turkaysoft** [reply]
Hello again! @wincmd64 

The Bluetooth Version Information module for Glow, which I believe will be one of the few apps to display Bluetooth information, is almost complete.

Stay tuned!

<img width="364" height="144" alt="image" src="https://github.com/user-attachments/assets/1f55108f-8d21-46c7-8430-ecb1d11ab9ff" />

**Reply by turkaysoft** [reply]
The suggestion has been implemented.

https://github.com/turkaysoftware/glow/releases/tag/v25.09
